packages/local_auth/local_auth_android/android/src/test/java/io/flutter/plugins/localauth/LocalAuthTest.java

```diff
@@ -61,4 +64,122 @@ public void onDetachedFromActivity_ShouldReleaseActivity() {
+        .thenReturn(BiometricManager.BIOMETRIC_ERROR_NO_HARDWARE);
+    plugin.setBiometricManager(mockBiometricManager);
+
+    plugin.onMethodCall(new MethodCall("getEnrolledBiometrics", null), mockResult);
+    verify(mockResult).success(Collections.emptyList());
+  }
+
+  @Test
+  public void getEnrolledBiometrics_shouldReturnEmptyList_withNoMethodsEnrolled() {
+    final LocalAuthPlugin plugin = new LocalAuthPlugin();
+    setPluginActivity(plugin, buildMockActivity());
+    final MethodChannel.Result mockResult = mock(MethodChannel.Result.class);
+    final BiometricManager mockBiometricManager = mock(BiometricManager.class);
+    when(mockBiometricManager.canAuthenticate())
+        .thenReturn(BiometricManager.BIOMETRIC_ERROR_NONE_ENROLLED);
+    plugin.setBiometricManager(mockBiometricManager);
+
+    plugin.onMethodCall(new MethodCall("getEnrolledBiometrics", null), mockResult);
+    verify(mockResult).success(Collections.emptyList());
+  }
+
+  @Test
+  public void getEnrolledBiometrics_shouldOnlyAddEnrolledBiometrics() {
+    final LocalAuthPlugin plugin = new LocalAuthPlugin();
+    setPluginActivity(plugin, buildMockActivity());
+    final MethodChannel.Result mockResult = mock(MethodChannel.Result.class);
+    final BiometricManager mockBiometricManager = mock(BiometricManager.class);
+    when(mockBiometricManager.canAuthenticate(BiometricManager.Authenticators.BIOMETRIC_WEAK))
+        .thenReturn(BiometricManager.BIOMETRIC_SUCCESS);
+    when(mockBiometricManager.canAuthenticate(BiometricManager.Authenticators.BIOMETRIC_STRONG))
+        .thenReturn(BiometricManager.BIOMETRIC_ERROR_NONE_ENROLLED);
+    plugin.setBiometricManager(mockBiometricManager);
+
+    plugin.onMethodCall(new MethodCall("getEnrolledBiometrics", null), mockResult);
+    verify(mockResult)
+        .success(
+            new ArrayList<String>() {
+              {
+                add("weak");
+              }
+            });
+  }
+
+  @Test
+  public void getEnrolledBiometrics_shouldAddStrongBiometrics() {
+    final LocalAuthPlugin plugin = new LocalAuthPlugin();
+    setPluginActivity(plugin, buildMockActivity());
+    final MethodChannel.Result mockResult = mock(MethodChannel.Result.class);
+    final BiometricManager mockBiometricManager = mock(BiometricManager.class);
+    when(mockBiometricManager.canAuthenticate(BiometricManager.Authenticators.BIOMETRIC_WEAK))
+        .thenReturn(BiometricManager.BIOMETRIC_SUCCESS);
+    when(mockBiometricManager.canAuthenticate(BiometricManager.Authenticators.BIOMETRIC_STRONG))
+        .thenReturn(BiometricManager.BIOMETRIC_SUCCESS);
+    plugin.setBiometricManager(mockBiometricManager);
+
+    plugin.onMethodCall(new MethodCall("getEnrolledBiometrics", null), mockResult);
+    verify(mockResult)
+        .success(
+            new ArrayList<String>() {
+              {
+                add("weak");
+                add("strong");
+              }
+            });
+  }
+
```

Comment:
The new `deviceSupportsBiometrics` needs tests (true and false cases).

Reply:
RIght, I completely forgot. They have been added.